index.js

```diff
@@ -20,7 +20,7 @@ module.exports = readDotEnvConfigs;
  * Application root directory.
  * @type {string}
  */
-var applicationRoot = require('app-root-path').toString();
+var applicationRoot = __dirname + '/../../';
```

Comment:
So you're assuming that loadenv's index.js will be in `node_modules/loadenv/index.js`? This may not be the case if they globally installed the library.